tests/e2e.py

```diff
@@ -73,3 +74,58 @@ async def test_import_document(self):
                 await client.delete_schema(schema.id)
             if workspace:
                 await client.delete_workspace(workspace.id)
+
+    async def test_create_upload(self):
```

Comment:
If you wanted to squash the commits before merge, I apologize for his unsolicited lecture :see_no_evil: 

This is one of the things I don't like about semantic release, it has a dedicated `test:` commit type which encourages people to add tests after the implementation happens which isn't a great Git practice -- tests should be part of the implementation to easily check both together during code reviews. They also serve as kind of a documentation which helps reviewers further. 

Commits should be atomic improvements of the codebase to be able to leverage the full power of Git. E.g. if you needed to revert the `retrieve_task` functionality, it wouldn't be a simple revert of one commit since the tests are part of a different one together with test for `create_upload`.

It's not worth fixing now but I just wanted to share my PoV on Git best practices :wink: 

Personally, I create `test:` commits only if I'm really adding tests for existing code.

Reply:
Addressed above.